Automattic/wp-calypso
# Launch: Cleanup calypsoifyGutenberg.

#### Changes proposed in this Pull Request

Note: Ensure ETK is deployed with changes from https://github.com/Automattic/wp-calypso/pull/49004 before deploying this one.

This PR branches off https://github.com/Automattic/wp-calypso/pull/49004 to cleanup props that are no longer used in `calypsoifyGutenberg`.

The reason for this PR's existence is to keep deploys atomic. https://github.com/Automattic/wp-calypso/pull/49004 is related to ETK and this is related to WBE.

~~TODO: The `currentCalypsoUrl` is the only thing left that keeps Focused Launch tethered to Calypso I think. Need to see how to move that out.~~ This is needed for for "Use A Domain I Own" to identify the correct url on the parent iframe.

These feature flags are also removed:
- gutenboarding/new-launch-mobile
- gutenboarding/feature-picker
- create/persistent-launch-button
- create/focused-launch-flow

#### Testing instructions



* Ensure Step-By-Step Launch & Focused Launch still works.
* Also good to check who is accessing props from `calypsoifyGutenberg` object, most places are still relying on `isGutenboarding` and that is why it is still kept.

Fixes https://github.com/Automattic/wp-calypso/issues/46839 #47803 #50089
